src/epics/addAssetToWatchlistEpic.js

```diff
@@ -44,7 +44,10 @@ const addAssetToWatchlistEpic = (action$, store, { client }) =>
       const normalizedList = listItems.map(val => {
         return { project_id: +val.project.id }
       })
-      const newListItems = [...normalizedList, { project_id: +projectId }]
+      const newListItems = projectId
+        ? [...normalizedList, { project_id: +projectId }]
+        : normalizedList
+
       const mutationPromise = client.mutate({
```

Comment:
We need to find a better name instead of mutationPromise. @DmitriVanGuard 
This name doesn't resolve what this method does.